### DISCUSSION TITLE
Welcome to fastify Discussions!

### DISCUSSION THREAD

**Discussion by mcollina** [fastify/fastify#2348] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ethan-Arrowood** [comment]
Woohoo! I'll kick it off:

Hello! My name is Ethan Arrowood, I'm a software engineer for Microsoft and one of the maintainers here on Fastify. I have contributed heavily in the TypeScript definitions, but have also worked on tests, docs, and some plugin maintenance.

To everyone, welcome to the Fastify community. Please feel free to reach out to me at any time about Fastify + TypeScript or if you're interested in contributing 😁

**Comment by jsumners** [comment]
👋 

I'm a backend engineer with a focus on libraries, frameworks, and tooling. I got involved with Fastify by way of my work on Pino. Basically, I just followed Matteo over 😆

These days, I mostly just review issues and PRs. Sometimes I find some free time to work on issues myself.

**Comment by Eomm** [comment]
I'm Manuel and I work as a backend developer to a tons of microservices built with Fastify.

The Fastify community and ecosystem teach me something now quite often 😀 and I have the opportunity to deep down tech aspect that would be difficult to see in my day-by-day at work like implement a plugin that adheres to an RFC spec! 😱

I work across all the Fastify org trying to solve more problems than I create 😇

**Comment by RafaelGSS** [comment]
Hey! I'm Rafael Gonzaga and I work as backend developer with a focus on performance optimization.

I use Fastify since 2.0 (don't know right about it), but for now I use it wrapped inside another framework. I try to help the community when I can.

Obs: Fastify already save us a lot of money :smile:
